pkg/operation/seed/istio/crds_test.go

```diff
@@ -86,11 +88,20 @@ var _ = Describe("#CRDs", func() {
 		ca := kubernetes.NewChartApplier(renderer, a)
 		Expect(ca).NotTo(BeNil(), "should return chart applier")
 
-		crd = NewIstioCRD(ca, chartsRootPath)
+		crd = NewIstioCRD(ca, chartsRootPath, c)
 
 	})
 
 	JustBeforeEach(func() {
+		deprectedCRDs := []apiextensionsv1.CustomResourceDefinition{
```

Comment:
Shouldn't v1beta1 be used for backwards compatibility/support of older kubernetes versions?